src/System.Drawing.Common/tests/Performance/Perf_Graphics_DrawBeziers.cs

```diff
@@ -11,7 +11,7 @@ namespace System.Drawing.Tests
     public class Perf_Graphics_DrawBeziers : RemoteExecutorTestBase
     {
         [Benchmark(InnerIterationCount = 10000)]
-        [ConditionalBenchmark(typeof(Helpers), nameof(Helpers.IsDrawingSupported))]
+        [ConditionalBenchmark(typeof(Helpers), nameof(Helpers.GetIsDrawingSupported))]
```

Comment:
I'm just curious, why does this fix it, if IsDrawingSupported just calls into GetIsDrawingSupported?

Reply:
Because the logic for perf condition benchmarks is a custom one (which I wrote some months ago to disable perf tests if gdi is not available) which can't handle this.